CODEOWNERS

```
@@ -119,6 +119,7 @@ k8s/**/common/rd/identity.yaml @hmcts/ref-data
 k8s/**/common/rd/professional-api.yaml @hmcts/ref-data
 k8s/**/common/rd/profile-sync.yaml @hmcts/ref-data
 k8s/**/common/rd/user-profile-api.yaml @hmcts/ref-data
+k8s/**/common/rd/judicial-api.yaml @hmcts/ref-data
```

this should be AAT until its approved for prod
```suggestion
k8s/aat/common/rd/judicial-api.yaml @hmcts/ref-data
```

---

Thanks, have amended